tkweb/apps/gallery/templates/image.html

```diff
@@ -51,6 +51,7 @@
 				src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
 				{% else %}
 				src="{{ file.file.thumbnail.720x720.url }}"
+				title="{{ file.caption }}"
```

Comment:
Burde det ikke stå uden for if'en?

Reply:
Jo, det giver da mere mening. Jeg har vist fikset det.